index.js

```diff
@@ -257,9 +257,7 @@ class Client {
             })
             .then(() => {
               this.serverless.cli.log(
-                `Success! Your site should be available at http://${bucketName}.${
-                  regionUrls[region]
-                }/`
+                `Success! Your site should be available at http://${bucketName}.${regionUrls[region]}/`
```

Comment:
Those changes (and the removed blank lines) seem to be introduced by the `husky` hook defined in `package.json`